app/components/record-button/template.hbs

```diff
@@ -1,7 +1,4 @@
-<div
-  class="record-button-container record-button-container--start-stop
-    {{if this.active 'recording'}}"
->
+<div class="record-button-container {{if this.active 'recording'}}">
```

Comment:
Let's use `record-button-container--recording` as active class since we mostly use BEM in our CSS: https://getbem.com/introduction/

Reply:
Is it now `BEM`-like enough? :upside_down_face: